mingw-w64-pdcursesmod/build.sh

```
@@ -20,10 +20,12 @@ else
 fi
 
 # install header files
-mkdir -p "$LIBRARY_PREFIX/include"
-for f in curses.h panel.h term.h; do
-    cp "$SRC_DIR/$f" "$LIBRARY_PREFIX/include/"
-done
+install -m 0644 curses.h panel.h term.h "$LIBRARY_PREFIX/include/pdcurses/"
+# needs defines matching the make step, see https://github.com/Bill-Gray/PDCursesMod/issues/133
+echo '#define PDC_WIDE 1'           >> pdcurses.h
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)